iree/base/internal/status.h

```diff
@@ -15,117 +15,303 @@
 #ifndef IREE_BASE_INTERNAL_STATUS_H_
 #define IREE_BASE_INTERNAL_STATUS_H_
 
-#include <atomic>
+#include <cstdint>
+#include <memory>
 #include <string>
 
-#include "absl/base/attributes.h"
-#include "absl/flags/flag.h"
 #include "absl/strings/string_view.h"
+#include "iree/base/api.h"
 #include "iree/base/internal/logging.h"
```

Comment:
Please change this to iree/base/logging.h now that this is unconditionally included in the build.

```suggestion
#include "iree/base/logging.h"
```